aws-s3-transfer-manager/Cargo.toml

```diff
@@ -44,7 +43,7 @@ fastrand = "2.1.1"
 futures-test = "0.3.30"
 md5 = "0.7.0"
 tempfile = "3.12.0"
-test-common = { path = "./test-common", version = "0.1.0" }
+test-common = { path = "./test-common" }
```

Comment:
hmmm, with version, cargo publish will fail https://github.com/rust-lang/cargo/issues/4242#issuecomment-2578428685
Without version, cargo deny will fail. https://github.com/EmbarkStudios/cargo-deny/issues/241
🤦‍♂️

Reply:
Maybe refactor the test common stuff so it's not a separate project with it's own `Cargo.toml` and instead is just in the `tests/` directory. See https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests

Reply:
Uh, that's a pain. Maybe it's time to fix `mock_client` in `aws_smithy_mocks_experimental` to properly stub out an HTTP client upstream. We have this `test-common` as a separate helper crate because `mock_client_with_stubbed_http_client` (a workaround for mock_client) are used both in unit tests and in integration tests.

If `mock_client` is fixed, then we can remove the `mock_client_with_stubbed_http_client` from `test-common` and move `drain` and `global_uuid_str` to a common module under `tests/common.rs`.

Reply:
Uh, that's a pain. Maybe it's time to fix `mock_client` in `aws_smithy_mocks_experimental` to properly stub out an HTTP client upstream. We have this `test-common` as a separate helper crate because `mock_client_with_stubbed_http_client` (a workaround for mock_client) are used both in unit tests and in integration tests.

If `mock_client` is fixed, then we can remove the `mock_client_with_stubbed_http_client` from `test-common`, move `drain` and `global_uuid_str` to a common module `tests/common.rs`, and remove the `test-common` crate.

Reply:
Maybe [allow-wildcard-paths](https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html?highlight=allow-wildcard-paths#the-allow-wildcard-paths-field-optional) can be used to unblock error from `cargo-deny`?

Reply:
Alternatively, maybe [allow-wildcard-paths](https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html?highlight=allow-wildcard-paths#the-allow-wildcard-paths-field-optional) can be used to unblock error from `cargo-deny`?